source/scripts/script.js

```diff
@@ -39,6 +44,7 @@ async function init() {
         if (event.key === 'Enter') {
             let searchSuccessful = await search();
             if (searchSuccessful) {
+                router.navigate(document.getElementById('search-query').value, false);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8436588)
```suggestion
                router.navigate(document.getElementById("search-query").value, false);
```